play-2.4/swagger-play2/build.sbt

```diff
@@ -8,10 +8,12 @@ crossScalaVersions := Seq("2.11.6", "2.11.7")
 
 libraryDependencies ++= Seq(
   "org.slf4j"          % "slf4j-api"       % "1.6.4",
-  "com.wordnik"       %% "swagger-jaxrs"   % "1.3.12",
+  "io.swagger"         % "swagger-core"   % "1.5.4",
+  "io.swagger" % "swagger-scala-module_2.11" % "1.0.0",
```

Comment:
Align formatting

Can you change `% "swagger-scala-module_2.11"` to `%% "swagger-scala-module"`